install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml

```diff
@@ -17,6 +17,10 @@ spec:
         strimzi.io/kind: cluster-operator
```

Comment:
Why are we switching 50 to 60? Are none of the numbers between 40-50 appropriate for the new CM?

Reply:
@scholzj ^

Reply:
I'd also argue `51` would also be appropriate for the CM (keeping deployment as `50`) as the CM should not need to be created prior to the deployment due to the dynamic file watcher?

Reply:
Well, we used different numbers for different types of resources so far. This really just continues.